packages/entities/entities-plugins/src/components/select/PluginSelectCard.vue

```diff
@@ -3,66 +3,78 @@
     :label="plugin.disabledMessage"
     position-fixed
   >
-    <KCard
-      class="plugin-card-content"
-      :class="{ disabled: isDisabled }"
+    <div
```

Comment:
question: why are we using a `div` as a `role="button"`? Can this be an `a` tag instead?

Reply:
I can switch it, @portikM had recommended `div` with `role="button"` in the associated ticket.

Reply:
I'd recommend an `a` anchor tag since they are natively focusable and clickable. Leave the `role="button"` unless we're actually navigating (e.g. if we're just opening a modal, or "selecting" something)